src/jdk.jdeps/share/classes/com/sun/tools/classfile/AccessFlags.java

```diff
@@ -60,7 +61,7 @@
     public static final int ACC_MANDATED      = 0x8000; //                          method parameter
     public static final int ACC_MODULE        = 0x8000; // class
 
-    public static enum Kind { Class, InnerClass, Field, Method}
+    public static enum Kind { Class, InnerClass, Field, Method, JavacExtra}
 
```

Comment:
This is an half hearted attempt to find a place for ACC_REF_DEFAULT - I don't think it belongs here really since it is not a part of the class flags. But I have stuck this in here for now - In javap code to reference ACC_REF_DEFAULT symbolically this is useful (since the compiler's definition from Flags.java is not visible there in javap)